Services/Migration/test/ilServicesMigrationSuite.php

```diff
@@ -0,0 +1,27 @@
+<?php declare(strict_types=1);
+
+/******************************************************************************
+ *
+ * This file is part of ILIAS, a powerful learning management system.
+ *
+ * ILIAS is licensed with the GPL-3.0, you should have received a copy
+ * of said license along with the source code.
+ *
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *
+ *****************************************************************************/
+
+use PHPUnit\Framework\TestSuite;
+
+class ilServicesMigrationSuite extends TestSuite
+{
+    public static function suite()
+    {
+        $suite = new ilServicesMigrationSuite();
+
+        return $suite;
+    }
+}
```

Comment:
`PSR-2 + X` ;-)

Reply:
Did you mean the return type? In this case i fix it. ;)

Reply:
I meant the missing `\n' at the end of the file.

Reply:
Ok, thank you. I added a new line, but github doesn't show this correctly.

Reply:
Looks good now ^^